src/app/shared/store/user.state.ts

```diff
@@ -76,25 +74,21 @@ export interface UserStateModel {
   selectedWorkshop: Workshop;
   selectedProvider: Provider;
   applications: Application[];
-  applicationsByStatus: Application[];
   children: Child[];
   favoriteWorkshops: Favorite[];
   favoriteWorkshopsCard: WorkshopCard[];
-  status: number;
 }
 @State<UserStateModel>({
   name: 'user',
   defaults: {
     isLoading: false,
-    workshops: Workshop[''],
+    workshops: [],
```

Comment:
how come?

Reply:
I've decided to keep the same way as more other default values 
![image](https://user-images.githubusercontent.com/27493959/132332984-de4d1b06-4e48-4364-bbe1-fc3a43c0607f.png)